### DISCUSSION TITLE
Special Interest Groups (SIGs)

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/OpenAPI-Specification#2695] [Announcements]
Following the launch of OpenAPI version 3.1.0, there were so many interesting topics for the community to explore that we created specific Special Interest Groups (SIGs) to help move forward on specific topics. Early SIGs include:
- Security
- Workflows
- Overlays
- Codegen
- Lifecycle
- Travel

The groups are self-organizing now, establishing their meeting cadence and on-boarding processes, so it is a great time to get involved. One easy way is to join the OpenAPI Initiative Slack channel. If you aren’t already a member, please go to  https://communityinviter.com/apps/open-api/openapi to get onboarded. Make sure to mention which SIG(s) you are interested in, and we look forward to having you as part of the discussion.

**Comment by handrews** [comment]
@earth2marsh do we want to keep announcements like this around forever, or should this one be closed as there's documentation on SIGs elsewhere now?

**Comment by handrews** [comment]
I'm going to make an executive decision given the lack of response and close this - we have SIG documentation in at least two repos right now anyway.  Please feel free to re-open if we want it to stay open permanently (and feel free to delete this and my previous comment suggesting closing it if that is the case).
